rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/core/RocketMQBatchListener.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.spring.core;
+
+import java.util.List;
+
+public interface RocketMQBatchListener<T> extends RocketMQListener<T> {
+
+    @Override
+    @Deprecated
+    default void onMessage(T message) {
+        throw new UnsupportedOperationException("It's a Unsupported API, please use onMessages method instead.");
+    }
+
+    /**
+     * Batch consume bulk Java Objects once.
+     *
+     * @param messages bulk Java Objects.
+     */
+    void onMessages(List<T> messages);
+}
```

Comment:
IMO, it would be better for RocketMQBatchListener to be an independent interface instead of inheriting from RocketMQListener.

Reply:
resolve to: https://github.com/apache/rocketmq-spring/pull/205